app/models/work_completion/labware_completion.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+# Shared behaviour of WorkCompletion::PlateCompletion and WorkCompletion::TubeCompletion
+#
+# @author Genome Research Ltd.
+#
+class WorkCompletion::LabwareCompletion
```

Comment:
👍 Very pleased to see this was possible.  Good work.

Reply:
Yes think it worked out well! Thanks for the suggestion.